code/datums/wires/mines.dm

```diff
@@ -0,0 +1,81 @@
+/datum/wires/mines
+	wire_count = 6
+	random = 1
+	holder_type = /obj/effect/mine
+
+var/const/WIRE_DETONATE  = 1
```

Comment:
Can you turn these into defines? It'll function identically, and just be handled by the compiler rather than having 6 more global vars floating around at runtime (Globals are generally bad unless they're exactly what you need for a certain situation)